### PR TITLE
[#169995597] added auto hide for alert after 10 seconds or on page change

### PR DESCRIPTION
**Pull request by d-al-ibm**
added timeout to automatically hide alert after 10 seconds
added auto hide for alerts on page change
fixed links with translations

### CODE REVIEW

File: src/components/AppAlert/AppAlert.tsx [hunk 1]
```diff
@@ -15,6 +15,21 @@ export const AppAlert = () => {
     alertContext.setAlert({ ...alertContext.alert, showAlert: false });
   };
 
+  useEffect(() => {
+    if (alertContext.alert.showAlert) {
```
Comment (gunzip) [inline src/components/AppAlert/AppAlert.tsx]: I suggest to replace the home made alert component with https://www.npmjs.com/package/react-alert
may you try with this PR ? (it has a timeout property)

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>This PR changes a total of 621 LOCs, that is more than a reasonable size of 250. Consider splitting the pull request into smaller ones.</td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#169995597](https://www.pivotaltracker.com/story/show/169995597): Come utente loggato, mi aspetto che i messaggi informativi di alert scompaiano automaticamente e nel caso di cambio di schermata o eventualmente dopo un certo periodo


New dependencies added: <a href='https://www.npmjs.com/package/react-alert'><code>react-alert</code></a> and <a href='https://www.npmjs.com/package/@types/react-alert'><code>&#64;types/react-alert</code></a>.


<h2><a href="https://github.com/schiehll/react-alert#readme">react-alert</a></h2>
<p>Author: Reinaldo Schiehll</p>
<p>Description: A simple react alert component</p>
<p>Homepage: <a href="https://github.com/schiehll/react-alert#readme">https://github.com/schiehll/react-alert#readme</a></p>

<table>
  <thead><tr><th></th><th width="100%"></th></tr></thead>
  <tr><td>Created</td><td>about 4 years ago</td></tr><tr><td>Last Updated</td><td>about 2 months ago</td></tr><tr><td>License</td><td>MIT</td></tr><tr><td>Maintainers</td><td>1</td></tr><tr><td>Releases</td><td>44</td></tr><tr><td>Direct Dependencies</td><td><a href='https://www.npmjs.com/package/@testing-library/jest-dom'><code>&#64;testing-library/jest-dom</code></a>, <a href='https://www.npmjs.com/package/@testing-library/react'><code>&#64;testing-library/react</code></a>, <a href='https://www.npmjs.com/package/prop-types'><code>prop-types</code></a> and <a href='https://www.npmjs.com/package/react-transition-group'><code>react-transition-group</code></a></td></tr><tr><td>Keywords</td><td>react, alert, react-alert, toaster, react-toaster and react-component</td></tr>
</table>
This README is too long to show.



<h2><a href="http://npmjs.com/package/@types/react-alert">&#64;types/react-alert</a></h2>
<p>Author: Unknown</p>
<p>Description: TypeScript definitions for react-alert</p>
<p>Homepage: <a href="http://npmjs.com/package/@types/react-alert">http://npmjs.com/package/@types/react-alert</a></p>

<table>
  <thead><tr><th></th><th width="100%"></th></tr></thead>
  <tr><td>Created</td><td>about 2 years ago</td></tr><tr><td>Last Updated</td><td>28 days ago</td></tr><tr><td>License</td><td>MIT</td></tr><tr><td>Maintainers</td><td>1</td></tr><tr><td>Releases</td><td>6</td></tr><tr><td>Direct Dependencies</td><td><a href='https://www.npmjs.com/package/@types/react'><code>&#64;types/react</code></a></td></tr>
</table>

<details>
<summary><code>README</code></summary></br>

# Installation
> `npm install --save @types/react-alert`

# Summary
This package contains type definitions for react-alert ( https://github.com/schiehll/react-alert ).

# Details
Files were exported from https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-alert

Additional Details
 * Last updated: Wed, 20 Mar 2019 17:54:34 GMT
 * Dependencies: @types/react
 * Global values: none

# Credits
These definitions were written by Yue Yang <https://github.com/g1eny0ung>.


</details>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 014df777740521351fb211506f331c1bb2b32b65
</p>

File: src/App.tsx [hunk 2]
```diff
@@ -13,16 +21,49 @@ import "./App.scss";
...
+        // fix for different names: bootstrap uses "danger", react-alert uses "error"
+        color={options.type === types.ERROR ? "danger" : options.type}
+        toggle={close}
+        className="mt-2 mb-0"
+      >
+        {message}
+      </Alert>
+    </div>
+  );
+
```
Comment (gunzip) [inline src/App.tsx]: can we isolate this code into a new file ? (ie. `AlertTemplate.tsx`)

File: src/components/DefaultContainer/DefaultContainer.tsx [hunk 3]
```diff
@@ -31,6 +30,8 @@ import { SlimHeader } from "../SlimHeader/SlimHeader";
 import { SpidLogin } from "../SpidLogin/SpidLogin";
 import { UserProfile as UserProfileComponent } from "../UserProfile/UserProfile";
 
+import "./AppAlert.css";
```
Comment (gunzip) [inline src/components/DefaultContainer/DefaultContainer.tsx]: this line belongs to `AlertTemplate.tsx`

**Pull request merged**